### ISSUE TITLE
Starting and trailing spaces "Fix String" button doesn't work correctly

### ISSUE THREAD

**Issue by tariver** [WeblateOrg/weblate#3150]
**Describe the bug**
"Fix string" buttons are not working correctly for "Starting spaces" and "Trailing spaces" checks.

**To Reproduce**
1. Have a source language strings with starting and trailing spaces. Mine is .NET resource file. Extract from the file (my source language is Russian):
```
  <data name="StartingSpace" xml:space="preserve">
    <value> Пробел в начале</value>
  </data>
  <data name="TrailingSpace" xml:space="preserve">
    <value>Пробел в конце </value>
  </data>
  <data name="BothSpaces" xml:space="preserve">
    <value> Пробел и в начале и в конце </value>
  </data>
```
2. Have a file with already translated strings to target language
Extract from the file (target language is English) (note that I didn't make any starting or trailing spaces on purpose):
```
  <data name="StartingSpace" xml:space="preserve">
    <value>Starting space</value>
  </data>
  <data name="TrailingSpace" xml:space="preserve">
    <value>Trailing space</value>
  </data>
  <data name="BothSpaces" xml:space="preserve">
    <value>Both starting and trailing spaces</value>
  </data>
```
3. Make a component from these files
4. Now when you go to component's English language section you have several failed checks of starting or/and trailing spaces.
5. Go to one of them and on the right you will see check dialogue boxes
![image](https://user-images.githubusercontent.com/36223356/67579325-e7cc1e80-f74c-11e9-8e7b-b94ad46d8cf7.png)
6. Press "Fix string" button on "Starting spaces" dialogue box
7. A _trailing_ space is added to the translation input field
8. Press "Fix string" button on "Starting spaces" dialogue box again
9. Another trailing space is added.
10. After this pressing "Fix string" button again repeatedly doesn't seem to do anything

Pressing "Fix string" in "Trailing space" dialogue box has the same effect - first time it adds a trailing space (which is fine), second time it adds another trailing space (not fine) and pressing button after this doesn't seem to have effect.
You can press "Fix string" first in one dialogue box then the other - the outcome is still the same.

This behaviour is observed on all of the three example strings.

If you just press the "Save" button (without pressing "Fix string" or after pressing them, doesn't matter) it just saves the translation with a whiteboard message that autofixes were applied. Checking back to the saved string you can see that autofixes were applied correctly (no bug there).

**Expected behavior**
Pressing "Fix string" button on "Starting spaces" dialogue box should add a space at the start of the string and pressing it after this should have no effect
Pressing "Fix string" in "Trailing space" dialogue box more than once should have no effect

**Server configuration and status**
 * Weblate 3.9-dev
 * Python 3.5.3
 * Django 2.2.5
 * Celery 4.3.0
 * celery-batches 0.2
 * six 1.12.0
 * social-auth-core 3.2.0
 * social-auth-app-django 3.1.0
 * django-appconf 1.0.3
 * translate-toolkit 2.4.0
 * translation-finder 1.6
 * Whoosh 2.7.4
 * defusedxml 0.6.0
 * Git 2.11.0
 * Pillow 6.1.0
 * python-dateutil 2.8.0
 * lxml 4.4.1
 * django-crispy-forms 1.7.2
 * django_compressor 2.3
 * djangorestframework 3.10.3
 * user-agents 2.0
 * jellyfish 0.7.2
 * diff-match-patch 20181111
 * ruamel.yaml 0.16.5
 * Mercurial 4.0
 * Database backends: django.db.backends.postgresql
 * Cache backends: default:RedisCache
 * Email setup: django.core.mail.backends.smtp.EmailBackend: localhost
 * Celery: memory://, N/A, eager
 * Platform: Linux 4.9.0-3-amd64 (x86_64)

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).

**Comment by tariver** [comment]
After changing the regular expression in chars.py file "Starting spaces" button started to work correctly, but "Trailing Space" button still adds another trailing space after pressing it second time.

**Comment by nijel** [comment]
Should be fixed by 7abeb64bac
